docker-compose.yaml

```diff
@@ -37,11 +37,21 @@ services:
   django:
     stdin_open: true
     tty: true
+    read_only: true
     build:
       context: .
       dockerfile: docker/DevDjangoDockerfile
       args:
         USERID: ${UID:?err}
+        DEPLOY_ENV: dev
+        DJANGO_ALLOWED_HOSTS: "app localhost"
+        DJANGO_DB_HOST: db
```

Comment:
Sooo I would avoid trying to import all these database env vars. We don't want the docker build to be dependent on having a database accessible. That means you'll have to do some re-organization of the django settings files or tweak the logic so that the database connection is optional in certain cases.